src/main/java/cz/tstrecha/timetracker/dto/mapper/TaskMapper.java

```diff
@@ -18,4 +20,6 @@ public interface TaskMapper {
     @Mapping(target = "createdAt", ignore = true)
     @Mapping(target = "nameSimple", expression = "java(StringUtils.stripAccents(task.getName()))")
     TaskEntity fromRequest(TaskCreateRequestDTO task, UserEntity user);
+
+    List<TaskDTO> toListDTO(List<TaskEntity> source);
```

Comment:
Not like that, reusibility:

`TaskDTO toDTO(TaskEntity source);`

and when calling, just do `entities.foreach(mapper::toDTO)`